lib/drivers/bayerContourNext.js

```diff
@@ -543,7 +545,7 @@ module.exports = function (config) {
             } else {
               recordType = result.recordType;
               // We only collect data records (R)
-              if (recordType === 'R') {
+              if (recordType === 'R' && result.timestamp) {
```

Comment:
@jh-bate You mean with the 'R'? No, it's specified in the spec to be a capital 'R' and should never be lowercase.